src/Npgsql/NpgsqlCommand.cs

```diff
@@ -598,7 +615,7 @@ Task Prepare(bool async, CancellationToken cancellationToken = default)
                 }
             }
 
-            _connectorPreparedOn = connector;
+            Log.Debug($"Preparing: {CommandText}", connector.Id);
```

Comment:
This is the slowest way of logging, not checking if the level is enabled, and not using the static logger pattern.

Reply:
Thanks @sebastienros, I'll clean this up (note that it's in explicit preparation, which is a path we don't use in our benchmarks).